specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerConnections.json

```diff
@@ -0,0 +1,323 @@
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Pending Scope Connections Sent From A Network Manager": {
+            "$ref": "./examples/NetworkManagerPendingScopeConnectionList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkManagerConnection": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkManagerConnectionProperties",
+          "description": "The scope connection properties"
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "./network.json#/definitions/SystemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The Network Manager Connection resource"
+    },
+    "NetworkManagerConnectionProperties": {
+      "type": "object",
+      "properties": {
+        "networkManagerId": {
+          "type": "string",
+          "description": "Network Manager Id."
+        },
+        "connectionState": {
+          "$ref": "./networkManagerScopeConnections.json#/definitions/ConnectionState"
+        }
+      },
+      "description": "Information about the network manager connection."
+    },
+    "NetworkManagerConnectionListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkManagerConnection"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next page of results."
+        }
+      },
+      "description": "List of network manager connections."
+    },
+    "PendingNetworkManagerConnectionListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkManagerConnection"
+          }
+        },
+        "skipToken": {
+          "type": "string",
+          "description": "When present, this value can be passed to a subsequent query call (together with the same query and scopes used in the current request) to retrieve the next page of data."
+        }
+      },
+      "description": "List of network manager connections."
+    }
+  },
+  "parameters": {
+    "ScopeParameter": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Scope id for subscription or management group.",
```

Comment:
Can you add an example of what the scope would look like. Take a look at the swagger for Policy Assignment. They have a similar construct